### DISCUSSION TITLE
Z offset changing on Auto home

### DISCUSSION THREAD

**Discussion by grrmisfit** [mriscoc/Ender3V2S1#31] [Q&A]
I think this might be a hardware issue but ive seen a few reports of it so might be a marlin issue. set your z offset then do auto home and more times then not it will change by +/- a few points. i hover around -2.74 to -2.89 and it will change between that number

**Comment by mriscoc** [comment]
Could be a backlash kind problem.

**Comment by pauluchin** [comment]
I performed the 1st z calibration and then performed home and recalibrated the z offset, leaving the latter well adjusted and without subsequent problems.
 It should be noted that I did it with a bed at 60 and a nozzle at 200, which are the temperatures that I normally use.
